tests/test_integration_8_days_snapshot_report/test_integration_8_days_snapshot_graph.py

```diff
@@ -0,0 +1,525 @@
+                            event_type=EventType.READY_TO_INTEGRATE_STATUSES.value,
+                            sendingSupplierName="EMIS",
+                            requestingSupplierName="TPP",
+                        )
+                    ),
+                    sourcetype="myevent",
+                )
+
+                # Integration
+                for outcome in integration_outcomes:
+                    index.submit(
+                        json.dumps(
+                            create_sample_event(
+                                conversation_id=f'integration_on_{idx}_with_{outcome}',
+                                registration_event_datetime=registration_time.strftime("%Y-%m-%dT%H:%M:%S%z"),
+                                event_type=EventType.EHR_RESPONSES.value,
+                                sendingSupplierName="EMIS",
+                                requestingSupplierName="TPP",
+                            )
+                        ),
+                        sourcetype="myevent",
+                    )
+
+                    index.submit(
+                        json.dumps(
+                            create_sample_event(
+                                conversation_id=f'integration_on_{idx}_with_{outcome}',
+                                registration_event_datetime=ehr_integration_datetime.strftime("%Y-%m-%dT%H:%M:%S%z"),
+                                event_type=EventType.EHR_INTEGRATIONS.value,
+                                sendingSupplierName="EMIS",
+                                requestingSupplierName="TPP",
+                                payload=create_integration_payload(
+                                    outcome=outcome
+                                )
+
+                            )),
+                        sourcetype="myevent"
+                    )
+
+            # Act
+            test_query = self.generate_splunk_query_from_report(
+                "gp2gp_integration_8_days_snapshot_report/gp2gp_integration_8_days_snapshot_report_overall_percentage"
+            )
+
+            test_query = set_variables_on_query(
+                test_query,
+                {
+                    "$index$": index_name,
+                    "$start_time$": report_start.strftime("%Y-%m-%dT%H:%M:%S%z"),
+                    "$end_time$": report_end.strftime("%Y-%m-%dT%H:%M:%S%z"),
+                    "$cutoff$": cutoff,
+                },
+            )
+
+            sleep(2)
+
+            telemetry = get_telemetry_from_splunk(
+                self.savedsearch(test_query), self.splunk_service
+            )
+            self.LOG.info(f"telemetry: {telemetry}")
+
+            # Assert
+            expected_values = {
+                "In flight": "5.56",
+                "Integrated on time": "27.78",
+                "Integrated after 8 days": "27.78",
+                "Not integrated after 8 days": "5.56",
+            }
```

Comment:
how many transfers are there in total. the percentages suggest 18 but finding it hard to count :)

Reply:
nm i see it now